common/prisma/index.ts

```diff
@@ -3,4 +3,6 @@ import { PrismaClient } from '@prisma/client';
 /* In the development environment, also log 'query'
    which is the SQL query sent to the database.
    Through that, queries can be optimized for performance */
-export const prisma = new PrismaClient(process.env.ENV !== 'dev' ? undefined : { log: ['query', 'info', `warn`, `error`] });
+export const prisma = new PrismaClient(
+    process.env.ENV !== 'dev' || process.env.LOG_FORMAT === 'brief' ? undefined : { log: ['query', 'info', `warn`, `error`] }
```

Comment:
I don't think that the log format should control which logs are published. Could we use a different environment for the integration tests?

Reply:
fair, added ENV=test

Reply:
huch habs ausversehen unresolved 😬